exporter/zipkinexporter/zipkin.go

```diff
@@ -171,8 +204,44 @@ func (ze *zipkinExporter) Start(host exporter.Host) error {
 func (ze *zipkinExporter) Shutdown() error {
 	ze.mu.Lock()
```

Comment:
it is likely that after other changes are done that this mutex can be removed, I actually don't see a need for it.